src/app/auth/Logout.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import cookie from "js-cookie";
+
+class Logout extends React.Component {
+  componentWillMount() {
+    cookie.remove("token");
+    cookie.remove("user");
+
+    this.props.history.push("/");
```

Comment:
how do you feel about using react-rouiter redirect instead of history.push?
https://tylermcginnis.com/react-router-programmatically-navigate/

Reply:
I don't feel _great_ about it but can adjust. It's probably a good idea.

Reply:
Just kidding. I like `Redirect` better 👍